token-syncer/src/token_syncer/spnego.clj

```diff
@@ -0,0 +1,40 @@
+;;
+;;       Copyright (c) 2017 Two Sigma Investments, LP.
+;;       All Rights Reserved
+;;
+;;       THIS IS UNPUBLISHED PROPRIETARY SOURCE CODE OF
+;;       Two Sigma Investments, LP.
+;;
+;;       The copyright notice above does not evidence any
+;;       actual or intended publication of such source code.
+;;
+(ns token-syncer.spnego
```

Comment:
Should we make our own spnego library and factor it out all places?

Reply:
+1 
We can have libraries for spnego, correlation-id, exception rendering, etc.

Reply:
Are we still planning on doing this?